### PR TITLE
Added Condition to check pdf file name length

### LINKED ISSUE #682: [Reorder Pages] Input value not accepted for large pdfs.
I have a pdf  of size 3 mb having around 549 pages. It fails to rearrange the pages. The file is named book.pdf

### Steps to reproduce
A medium sized pdf will do.

### Expected behaviour
Should rearrange.

Input value is not valid error
image

### PR DESCRIPTION
**Pull request by BismeetSingh**
# Description
Limited file pages length to 50.

Fixes #682 

## Type of change
Just put an x in the [] which are valid.
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [x] `./gradlew assembleDebug assembleRelease`
- [ ] `./gradlew checkstyle`

# Checklist:
-[x] My code follows the style guidelines of this project
-[x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x ] My changes generate no new warnings

### CODE REVIEW

**Comment by Swati4star** [comment]
you are not supposed to merge Pull Requests. 
Look into resolving merge conflicts (You can google them)

**Comment by BismeetSingh** [comment]
Okay, I will try to figure that out.

File: app/src/main/java/swati4star/createpdf/activity/PreviewActivity.java [hunk 1]
```diff
@@ -134,7 +134,7 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
                     mImagesArrayList = data.getStringArrayListExtra(Constants.RESULT);
                     mPreviewAdapter.setData(mImagesArrayList);
                     mViewPager.setAdapter(mPreviewAdapter);
-                } catch (Exception e) {
+                }  catch (Exception e) {
```
[the remark this answers is not in the document]
Comment (vyankatesh24) [reply app/src/main/java/swati4star/createpdf/activity/PreviewActivity.java]: revert this change!

File: app/src/main/java/swati4star/createpdf/fragment/RemovePagesFragment.java [hunk 2]
```diff
@@ -106,6 +97,14 @@
     TextView mCompressionInfoText;
     @BindView(R.id.view_pdf)
     Button mViewPdf;
+    private Activity mActivity;
```
[the remark this answers is not in the document]
Comment (vyankatesh24) [reply app/src/main/java/swati4star/createpdf/fragment/RemovePagesFragment.java]: revert this refactoring changes!

File: app/src/main/java/swati4star/createpdf/fragment/RemovePagesFragment.java [hunk 3]
```diff
@@ -250,6 +266,7 @@ private void compressPDF() {
             } else {
                 String outputPath = mPath.replace(mActivity.getString(R.string.pdf_ext),
                         "_edited" + check + mActivity.getString(R.string.pdf_ext));
+
```
Comment (vyankatesh24) [inline app/src/main/java/swati4star/createpdf/fragment/RemovePagesFragment.java]: revert this change!

**Comment by vyankatesh24** [comment]
Congratulations! @BismeetSingh on your first contribution to project Images To PDF :+1:
